index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge,chrome=1">
+    <meta charset="utf-8">
+    <meta name="author" content="Aleksandr Schemelev">
+    <meta name="keywords" content="portfolio, Web Developing, simple HTML page, RSSchool padawan">
+    <link rel="stylesheet" href="css/style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Catamaran" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Black+Ops+One" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Special+Elite" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script" rel="stylesheet">
+    <title>Aleksandr Schemelev</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <a href="index.html" class="header-caption">Aleksandr Schemelev</a>
+        <nav>
+          <ul id="header">
+            <li><a href="#about-me">About Me</a></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#contacts">Contacts</a></li>
+          </ul>
+        </nav>
+      </header>
+      <main>
+        <div class="wrapper-main">
+          <div class="about-me">
+            <h2 id="about-me">Road to WEB Developing</h2>
+            <p>Ave! <a href="https://youtu.be/RQqF38SsmNM?t=4172" target="_blank"><img src="image/me.jpg" alt="Historical moment" 
```

Comment:
Img tag should not be placed in the middle of the text in such cases. And there is no need in inline styles here, it should be in css file